common/src/main/java/tds/common/configuration/RestTemplateConfiguration.java

```diff
@@ -1,14 +1,29 @@
 package tds.common.configuration;
 
-import org.springframework.boot.web.client.RestTemplateBuilder;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.client.BufferingClientHttpRequestFactory;
+import org.springframework.http.client.SimpleClientHttpRequestFactory;
 import org.springframework.web.client.RestTemplate;
 
+import java.util.Arrays;
+
+import tds.common.web.interceptors.RestTemplateLoggingInterceptor;
+
 @Configuration
+@Import(JacksonObjectMapperConfiguration.class)
 public class RestTemplateConfiguration {
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
     @Bean
-    public RestTemplate restTemplate(RestTemplateBuilder builder) {
-        return builder.build();
+    public RestTemplate restTemplate() {
```

Comment:
The way I read the documentation is that what you're doing is replacing the auto configured rest template with your own version.  So we are losing the ability to get the auto configured version.